docs/source/notebooks/probabilistic_matrix_factorization.ipynb

```diff
@@ -22,9 +22,10 @@
    ],
```

Comment:
<p>Isn't <code>time</code> a built-in import? If such, I think this should be place before <code>theano</code>  and <code>scipy</code>,  in the same block as <code>logging</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4074/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>